src/api/status/public/index.html

```diff
@@ -1224,5 +1244,6 @@ <h6 class="mt-3">Thank you for sharing!</h6>
     <script src="assets/js/demo-sidenav.js"></script>
     <!-- Control Center for Material Dashboard: parallax effects, scripts for the example pages etc -->
     <script src="assets/js/material-dashboard.min.js?v=3.0.0"></script>
+    <script src="postsCount.js"></script>
```

Comment:
Branch is outdated. There should be `/public/js` for js files.

Reply:
Yeah I think we have `index.js` under /public/js/feed

Reply:
Hi, how can I fix this issue? I don't really understand it

Reply:
What I would do is that I will include `script.js` in the `index.html` and have it call the `setPostCount` or whatever it is and `setPostCount` should be placed in `index.js` which is under `/public/js/post`

Reply:
@nguyenhung15913 The current master branch has a directory for js files, you should rebase your beach.

![image](https://user-images.githubusercontent.com/52666982/142279797-8f16ec0c-4873-4d66-a6f9-c46bdd1e9699.png)